sdcm/nemesis.py

```diff
@@ -1434,7 +1434,7 @@ def _parse_toppartitions_output(output):
             pattern1 = r"(?P<sampler>[A-Z]+)\sSampler:\W+Cardinality:\s~(?P<cardinality>[0-9]+)\s\((?P<capacity>[0-9]+)\scapacity\)\W+Top\s(?P<toppartitions>[0-9]+)\spartitions:"
             pattern2 = r"(?P<partition>[\w:]+)\s+(?P<count>[\d]+)\s+(?P<margin>[\d]+)"
             toppartitions = {}
-            for out in output.split('\n\n'):
+            for out in output.strip().split('\n\n'):
                 partition = OrderedDict()
                 sampler_data = re.match(pattern1, out, re.MULTILINE)
                 sampler_data = sampler_data.groupdict()
```

Comment:
Can you please add a None check + assertion in case regex didn't find anything

Reply:
added.